projects/core/src/lib/mask.ts

```diff
@@ -306,7 +306,7 @@ export class Maskito extends MaskHistory {
             return event.preventDefault();
         }
 
-        const [from, to] = elementState.selection;
+        const [from, to] = initialElementState.selection;
         const newPossibleValue =
             initialElementState.value.slice(0, from) +
```

Comment:
Fix only logic inside `handleInsert` because `handleDelete` already has the valid one

https://github.com/taiga-family/maskito/blob/218789d643de5da1f2d29accc50ccd8406a4f567/projects/core/src/lib/mask.ts#L268-L270